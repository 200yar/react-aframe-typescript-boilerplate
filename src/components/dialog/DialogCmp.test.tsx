import React from 'react';
import ReactDOM from 'react-dom';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import {shallowToJson} from "enzyme-to-json";
import Dialog from "./DialogCmp";
Enzyme.configure({ adapter: new Adapter() });

describe('Dialog Component', () => {

  const component = shallow(<Dialog>Dummy content</Dialog>);

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Dialog />, div);
  });

  it('matches snapshot', () => {
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});