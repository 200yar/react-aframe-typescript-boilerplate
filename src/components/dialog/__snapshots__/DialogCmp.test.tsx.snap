// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog Component matches snapshot 1`] = `
<div
  className="dialogContainer"
>
  <div
    className="dialogContent scale-up"
  >
    <div
      className="dialog-title"
    />
    Dummy content
  </div>
</div>
`;
